packages/grid/test/frozen-columns-polymer.test.js

```diff
@@ -0,0 +1,2 @@
+import '../all-imports.js';
```

Comment:
The `frozen-columns.test.js` only imports `vaadin-grid.js`, no need to use `all-imports.js` here.

Reply:
Done